sktime/forecasting/fbprophet.py

```diff
@@ -138,6 +140,7 @@ def __init__(
         freq=None,
         add_seasonality=None,
         add_country_holidays=None,
+        fit_kwargs=None,
```

Comment:
We should add it at the end, because here it is a breaking change that would need deprecation.
Why: suppose a user uses args, not kwargs, in the constructor.